modules/exploits/unix/webapp/pfadmin_protected_alias.rb

```diff
@@ -0,0 +1,141 @@
+
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
I'll first change all the other remarks and move the code afterwards.  
Does this also effect the code under `lib/msf/core/` ?